### PR TITLE
I18N, dependencies, dayjs and eslint

### PR DESCRIPTION
**Pull request by Antosik**
**Done**:
- Flatpickr internationalization;
- Use [dayjs](https://day.js.org/) instead of [momentjs](https://momentjs.com/);
- Remove unused bootstrap-datetimepicker.min.js;
- Update dependencies;
- Eslint added;
- Russian localization update

### CODE REVIEW

**Comment by Antosik** [comment]
@barisusakli, @julianlam, @psychobunny 
Can someone review this? Or maybe I can split this big PR into small PRs that can be easier to review 🤔

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "commonjs": true,
+        "es2021": true,
+        "node": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "rules": {
+    }
+}
```
Comment (julianlam) [inline .eslintrc:13]: Can we actually inherit the `.eslintrc` from NodeBB core just to be consistent? :smile:

File: package.json [hunk 2]
```diff
@@ -21,11 +22,13 @@
     "url": "https://github.com/Schamper/nodebb-plugin-poll/issues"
   },
   "dependencies": {
-    "async": "~1.5.2",
-    "xregexp": "~3.1.0",
```
Comment (julianlam) [inline package.json:25]: Mm, I see this was removed, where was it used, if at all?

**Review by julianlam — CHANGES_REQUESTED** [review]

**Comment by julianlam** [comment]
Functionality test pending

**Comment by gasoved** [comment]
@julianlam LGTM, we can add eslint rules from core.

File: package.json (hunk 2 above)
Comment (gasoved) [reply package.json:25]: It wasn't used at all.

**Pull request merged**